src/views/Orders.vue

```diff
@@ -212,6 +226,25 @@ export default defineComponent({
       });
       return batchmodal.present();
     },
+    async deleteBatch(batch: any) {
+      const deleteBatchAlert = await alertController
+        .create({
+          message: this.$t('Are you sure?'),
```

Comment:
Did not use correct alert here like we discussed.

Reply:
Reference:
https://github.com/hotwax/job-manager/blob/main/src/views/Pipeline.vue#L446

Reply:
> Reference:
> https://github.com/hotwax/job-manager/blob/main/src/views/Pipeline.vue#L446

Thank you sir! I am referring this link.